CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+# 4.2.1
+Technical improvement.
```

Comment:
So needs version bump in setup.py. Would be neat if we didn't repeat the version number, and/or generally automated the version bumps.